bottlerocket/samples/Makefile.toml

```diff
@@ -0,0 +1,242 @@
+[config]
+skip_core_tasks = true
+
+[env]
+AGENT_IMAGE_REGISTRY = { value = "public.ecr.aws/bottlerocket-test-system", condition = { env_not_set = ["AGENT_IMAGE_REGISTRY"] } }
+AGENTS = ["eks-resource-agent"]
+METADATA_BASE_URL = { value = "https://updates.bottlerocket.aws/2020-07-07", condition = { env_not_set = ["METADATA_BASE_URL"] } }
+ARCHITECTURE = { value = "x86_64", condition = { env_not_set = ["ARCHITECTURE"] } }
+CLUSTER_TYPE = { value = "eks", condition = { env_not_set = ["CLUSTER_TYPE"] } }
+ASSUME_ROLE = { value = "~", condition = { env_not_set = ["ASSUME_ROLE"] } }
+AWS_REGION = { value = "us-west-2", condition = { env_not_set = ["AWS_REGION"] } }
+UPGRADE_VERSION = { value = "v1.11.1", condition = { env_not_set = ["UPGRADE_VERSION"] } }
+STARTING_VERSION = { value = "v1.11.0", condition = { env_not_set = ["STARTING_VERSION"] } }
+SONOBUOY_MODE = { value = "quick", condition = { env_not_set = ["SONOBUOY_MODE"] } }
+TARGETS_URL = { value = "https://updates.bottlerocket.aws/targets", condition = { env_not_set = ["TARGETS_URL"] } }
+K8S_VERSION = { value = "1.24", condition = { env_not_set = ["K8S_VERSION"] } }
+
+# WORKLOAD_TEST_NAME = 
+# WORKLOAD_TEST_IMAGE_URI = 
+# CONTROL_PLANE_ENDPOINT_IP = 
+# MGMT_CLUSTER_KUBECONFIG_PATH = 
+# GOVC_USERNAME = 
+# GOVC_PASSWORD = 
+# GOVC_DATACENTER = 
+# GOVC_DATASTORE = 
+# GOVC_URL = 
+# GOVC_NETWORK = 
+# GOVC_RESOURCE_POOL = 
+# GOVC_FOLDER = 
```

Comment:
?

Reply:
These are variables that need to be set by the user if a workload test file is being generated (the first two variables) or if a vmware file is being generated (the other variables) but I haven't found a way yet to convey that to the user. Might move them into a `help` target. This location is just temporary.

Reply:
@mjsterckx Can you add a comment at the top explaining that these are optional and provide comments how they should be filled.